orodha-keycloak/orodha_keycloak/connection.py

```diff
@@ -0,0 +1,162 @@
+"""
+This Module contains the KeycloakConnection class which is a facade
+used to interact with a keycloak server via python-keycloak.
+"""
+import orodha_keycloak.connections.admin
+import orodha_keycloak.connections.client
+from orodha_keycloak.connections.exceptions import InvalidConnectionException
+
+
+class KeycloakConnection:
```

Comment:
@Edison-Stuart Up to you, but I might suggest re-naming this to `OrodhaKeycloakClient` as there are already `xxxKeycloakxxx` items floating around in the namespace from `python-keycloak`.

Reply:
This makes sense to me. I changed the class to OrodhaKeycloakClient as per your suggestion.